### ISSUE TITLE
Test if accessibility criteria are met

### ISSUE THREAD

**Issue by fhauptmann** [SORMAS-Foundation/SORMAS-Project#2379]
### Situation Description
German laws demand that from September 2020 all web based tools used by german authorities need to be accessible by handicapped people (barrierefrei). In future a certificate proving that all criteria are met, will be needet. 
### Feature Description
As a first step some simple tests to determine, if the accessibility criteria are met should be conducted: 

https://bik-fuer-alle.de/easy-checks.html
https://testen.bitv-test.de/selbstbewertung/index.php

**Comment by Marc-FS** [comment]
Any further information on the implementation of this feature?

**Comment by Candice-Louw** [comment]
Hallo @Marc-FS - thank you for raising this important topic!  An update from our side: an internal assessment on the current stance is planned to start during KW40.  Subsequent follow-up activities will then follow in a sequential manner, based on the results of the assessment.
